### ISSUE TITLE
Likelihood of a single inbred individual

### ISSUE THREAD

**Issue by magnusdv** [mkruijver/symbolicpedprob#1]
When only one individual is typed, `sLikelihood()` currently simply returns the Hardy-Weinberg expressions, without correcting for inbreeding:

``` r
library(symbolicpedprob)
library(pedtools)

x = cousinPed(1, child = T) |> 
  addMarker(`9` = "a/a", alleles = c("a", "b"))

sLikelihood(x)
#> [1] "a^2"

# Expected 1/16*a + 15/16*a^2
```

<sup>Created on 2023-09-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by mkruijver** [comment]
Good catch! I have now worked around this by calling the `ribd::inbreeding` function and manually adjusting the likelihood. (https://github.com/mkruijver/symbolicpedprob/commit/b13b23b95d371de724aa3759f0a81bb8a65158cd). I've also added a couple of tests to verify the results against the text-book case you mentioned above and a manual calculation using `pedprobr` (https://github.com/mkruijver/symbolicpedprob/commit/3a5ab8c2236498a867fb1b8d3518ed8fc9a262ee).

Note that these calculations will need to be updated if `theta>0` is implemented in the `sLikelihood` function.
